protos/opts/ps_opts_read.proto

```diff
@@ -79,6 +84,9 @@ message ReadOptions {
     // @gotags: kong:"help='Convert output before it is printed to STDOUT (options: unset, base64, gzip)',type=pbenum,pbenum_strip_prefix=CONVERT_OPTION_,pbenum_lowercase,default=unset"
     ConvertOption convert_output = 6;
 
+    // @gotags: kong:"-"
+    ReadFilterOptions read_filter = 7;
```

Comment:
Less stuttering when referencing this. `readOptions.Filter` vs `readOptions.ReadFilter`

```suggestion
    ReadFilterOptions filter = 7;
```